src/BlazorWebView/src/Maui/Microsoft.AspNetCore.Components.WebView.Maui.csproj

```diff
@@ -15,7 +15,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Compile Include="..\SharedSource\**\*.cs" Link="Windows\SharedSource\%(Filename)%(Extension)"/>
+    <Compile Include="..\SharedSource\**\*.cs" Link="Windows\SharedSource\%(Filename)%(Extension)" />
+    <Compile Include="..\..\..\Core\src\TaskExtensions.cs" Link="Utilities\TaskExtensions.cs" />
```

Comment:
I should just be able to use the `TaskExtensions` however they aren't being supported in the net6.0-android context. Added a file reference instead.